salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -325,8 +325,29 @@ server {
     }
     {% endif -%}
 
+    {% if pillar.journal.privacy_policy_url %}
```

Comment:
Ah! seen it: should be `privacy_notice_url`

Reply:
Oh dear, so it wasn't the first letter I typed mistakenly.